src/main/java/mate/academy/bookshop/repository/book/AuthorSpecificationProvider.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.bookshop.repository.book;
+
+import java.util.Arrays;
+import mate.academy.bookshop.model.Book;
+import mate.academy.bookshop.repository.specification.SpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthorSpecificationProvider implements SpecificationProvider<Book> {
+    @Override
+    public Specification<Book> getSpecification(String[] params) {
+        return (root, query, criteriaBuilder) -> root.get("author")
```

Comment:
create a constant for the parameter and reuse it. Better make it public and also use it in BookSpecificationBuilder